<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Simulator</class>
 <widget class="QWidget" name="Simulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1905</width>
    <height>874</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1831</width>
     <height>711</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QCheckBox" name="checkBox2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox {
    spacing: 0; /* 去掉间距 */
    min-width: 100px;
    min-height: 100px;
    background-image: url(:/Resources/Images/uncheck.png);
    background-repeat: no-repeat;
    background-position: center;
    background-size: cover; /* 或者用 contain，确保图片填满 */
    border: none; /* 去掉边框，防止影响点击区域 */
}

QCheckBox::indicator {
    width: 0px;
    height: 0px; /* 隐藏默认指示器 */
}

QCheckBox:checked {
    background-image: url(:/Resources/Images/check.png);
}
QCheckBox {
    text-align: center; /* 确保文本居中 */
    color: transparent; /* 将文本颜色设置为透明 */
}</string>
      </property>
      <property name="text">
       <string>模拟器2</string>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="9">
     <spacer name="horizontalSpacer_18">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="2">
     <widget class="QCheckBox" name="checkBox8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox {
    spacing: 5px;
    min-width: 100px;
    min-height: 100px;
    background-image: url(:/Resources/Images/uncheck.png);
    background-repeat: no-repeat;
    background-position: center;
}

QCheckBox::indicator {
    width: 0px;
    height: 0px;
}

QCheckBox:checked {
    background-image: url(:/Resources/Images/check.png);
}
QCheckBox {
    text-align: center; /* 确保文本居中 */
    color: transparent; /* 将文本颜色设置为透明 */
}</string>
      </property>
      <property name="text">
       <string>模拟器8</string>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QCheckBox" name="checkBox4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox {
    spacing: 5px;
    min-width: 100px;
    min-height: 100px;
    background-image: url(:/Resources/Images/uncheck.png);
    background-repeat: no-repeat;
    background-position: center;
}

QCheckBox::indicator {
    width: 0px;
    height: 0px;
}

QCheckBox:checked {
    background-image: url(:/Resources/Images/check.png);
}
QCheckBox {
    text-align: center; /* 确保文本居中 */
    color: transparent; /* 将文本颜色设置为透明 */
}</string>
      </property>
      <property name="text">
       <string>模拟器4</string>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QCheckBox" name="checkBox3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox {
    spacing: 5px;
    min-width: 100px;
    min-height: 100px;
    background-image: url(:/Resources/Images/uncheck.png);
    background-repeat: no-repeat;
    background-position: center;
}

QCheckBox::indicator {
    width: 0px;
    height: 0px;
}

QCheckBox:checked {
    background-image: url(:/Resources/Images/check.png);
}
QCheckBox {
    text-align: center; /* 确保文本居中 */
    color: transparent; /* 将文本颜色设置为透明 */
}</string>
      </property>
      <property name="text">
       <string>模拟器3</string>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer_15">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="5">
     <spacer name="horizontalSpacer_26">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="6">
     <widget class="QCheckBox" name="checkBox10">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox {
    spacing: 5px;
    min-width: 100px;
    min-height: 100px;
    background-image: url(:/Resources/Images/uncheck.png);
    background-repeat: no-repeat;
    background-position: center;
}

QCheckBox::indicator {
    width: 0px;
    height: 0px;
}

QCheckBox:checked {
    background-image: url(:/Resources/Images/check.png);
}
QCheckBox {
    text-align: center; /* 确保文本居中 */
    color: transparent; /* 将文本颜色设置为透明 */
}</string>
      </property>
      <property name="text">
       <string>模拟器10</string>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <spacer name="verticalSpacer_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer_14">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="10">
     <spacer name="verticalSpacer_8">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="5">
     <spacer name="horizontalSpacer_16">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="4">
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="10">
     <widget class="QCheckBox" name="checkBox12">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox {
    spacing: 5px;
    min-width: 100px;
    min-height: 100px;
    background-image: url(:/Resources/Images/uncheck.png);
    background-repeat: no-repeat;
    background-position: center;
}

QCheckBox::indicator {
    width: 0px;
    height: 0px;
}

QCheckBox:checked {
    background-image: url(:/Resources/Images/check.png);
}
QCheckBox {
    text-align: center; /* 确保文本居中 */
    color: transparent; /* 将文本颜色设置为透明 */
}</string>
      </property>
      <property name="text">
       <string>模拟器12</string>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <spacer name="horizontalSpacer_25">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="3">
     <spacer name="horizontalSpacer_27">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="8">
     <widget class="QCheckBox" name="checkBox11">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox {
    spacing: 5px;
    min-width: 100px;
    min-height: 100px;
    background-image: url(:/Resources/Images/uncheck.png);
    background-repeat: no-repeat;
    background-position: center;
    background-size: contain; /* 使用contain属性使图像适应按钮 */
}

QCheckBox::indicator {
    width: 0px;
    height: 0px;
}

QCheckBox:checked {
    background-image: url(:/Resources/Images/check.png);
    background-repeat: no-repeat;
    background-position: center;
    background-size: contain; /* 使用contain属性使图像适应按钮 */
}
/*QCheckBox {
    ...
    background-position-x: center;
    background-position-y: center;
    background-size-x: 100%;
    background-size-y: 100%;
}*/
QCheckBox {
    text-align: center; /* 确保文本居中 */
    color: transparent; /* 将文本颜色设置为透明 */
}</string>
      </property>
      <property name="text">
       <string>模拟器11</string>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="8">
     <spacer name="verticalSpacer_7">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="4">
     <widget class="QCheckBox" name="checkBox9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox {
    spacing: 5px;
    min-width: 100px;
    min-height: 100px;
    background-image: url(:/Resources/Images/uncheck.png);
    background-repeat: no-repeat;
    background-position: center;
}

QCheckBox::indicator {
    width: 0px;
    height: 0px;
}

QCheckBox:checked {
    background-image: url(:/Resources/Images/check.png);
}
QCheckBox {
    text-align: center; /* 确保文本居中 */
    color: transparent; /* 将文本颜色设置为透明 */
}</string>
      </property>
      <property name="text">
       <string>模拟器9</string>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <spacer name="horizontalSpacer_17">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="9">
     <spacer name="horizontalSpacer_24">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="8">
     <widget class="QCheckBox" name="checkBox5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox {
    spacing: 5px;
    min-width: 100px;
    min-height: 100px;
    background-image: url(:/Resources/Images/uncheck.png);
    background-repeat: no-repeat;
    background-position: center;
}

QCheckBox::indicator {
    width: 0px;
    height: 0px;
}

QCheckBox:checked {
    background-image: url(:/Resources/Images/check.png);
}
QCheckBox {
    text-align: center; /* 确保文本居中 */
    color: transparent; /* 将文本颜色设置为透明 */
}</string>
      </property>
      <property name="text">
       <string>模拟器5</string>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="10">
     <widget class="QCheckBox" name="checkBox6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox {
    spacing: 5px;
    min-width: 100px;
    min-height: 100px;
    background-image: url(:/Resources/Images/uncheck.png);
    background-repeat: no-repeat;
    background-position: center;
}

QCheckBox::indicator {
    width: 0px;
    height: 0px;
}

QCheckBox:checked {
    background-image: url(:/Resources/Images/check.png);
}
QCheckBox {
    text-align: center; /* 确保文本居中 */
    color: transparent; /* 将文本颜色设置为透明 */
}</string>
      </property>
      <property name="text">
       <string>模拟器6</string>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <spacer name="horizontalSpacer_28">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QCheckBox {
    spacing: 5px;
    min-width: 100px;
    min-height: 100px;
    background-image: url(:/Resources/Images/uncheck.png);
    background-repeat: no-repeat;
    background-position: center;
}

QCheckBox::indicator {
    width: 0px;
    height: 0px;
}

QCheckBox:checked {
    background-image: url(:/Resources/Images/check.png);
}
QCheckBox {
    text-align: center; /* 确保文本居中 */
    color: transparent; /* 将文本颜色设置为透明 */
}</string>
      </property>
      <property name="text">
       <string>模拟器7</string>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">QWidget {
    background-image: url(:/images/check.png);
    background-repeat: no-repeat;
    background-position: center;
    background-size: cover; /* 或者使用 contain，根据需要选择 */
}
</string>
      </property>
      <widget class="QCheckBox" name="checkBox1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>281</width>
         <height>331</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox {
    spacing: 5px;
    min-width: 100px;
    min-height: 100px;
    background-image: url(:/Resources/Images/uncheck.png);
    background-repeat: no-repeat;
    background-position: center;
    background-size: cover; /* 使图像覆盖整个按钮 */
}

QCheckBox::indicator {
    width: 0px;
    height: 0px;
}

QCheckBox:checked {
    background-image: url(:/Resources/Images/check.png);
    background-repeat: no-repeat;
    background-position: center;
    background-size: cover; /* 使图像覆盖整个按钮 */
}
QCheckBox {
    text-align: center; /* 确保文本居中 */
    color: transparent; /* 将文本颜色设置为透明 */
}</string>
       </property>
       <property name="text">
        <string>模拟器1</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
